nodejs/week1/src/backend/routes/meals.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const filepath = require('path').resolve(__dirname, '..');
+
+
+function getReview(id) {
+    let revData = fs.readFileSync(`${filepath}/data/reviews.json`);
+    let reviews = JSON.parse(revData);
+    const reviewArray = [];

```

Comment:
This is purely personal taste and conventions you/your team are using but consider using more functional programming approach - it's also shorter which is always cool if you can maintain readability

```suggestion
    const reviewArray = reviews.filter(review => review.id === id);
```